casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -376,12 +374,12 @@ object RuntimeManager {
       _                <- active.space.clear()
       _                <- active.replaySpace.clear()
       _                <- Runtime.injectEmptyRegistryRoot(active.space, active.replaySpace)
+      _                <- Runtime.bootstrapRegistry(active)
```

Comment:
can you rename `active` to `runtime` on the occasion? :)

Reply:
Done